src/Console/Command/Compile.php

```diff
@@ -852,4 +867,30 @@ private function logEndBuilding(CompileLogger $logger, SymfonyStyle $io, Box $bo
             )
         );
     }
+
+    private function renderMessages(Configuration $config, SymfonyStyle $io): void
+    {
+        $recommendations = $config->getRecommendations();
+        $warnings = $config->getWarnings();
+
+        if ([] === $recommendations) {
```

Comment:
I guess you're not using `doctrine/coding-standard` (doesn't allow yoda-style) ;)

Reply:
PHP CS Fixer configuration